torchchat/generate.py

```diff
@@ -105,6 +122,9 @@ class GeneratorArgs:
         None  # When passed into the Generator, this will be used as the system prompt
     )
     encoded_prompt: Optional[torch.Tensor] = None
+    image_prompts: Optional[List[Union[str, PathLike, bytes]]] = (
```

Comment:
Let's use sequence instead of List for better clarity:

```suggestion
    image_prompts: Optional[Sequence[Union[str, PathLike, bytes]]] = (
```